pydicom/dataelem.py

```diff
@@ -329,41 +329,56 @@ def from_json(cls, dataset_class, tag, vr, value, value_key,
                     # Some DICOMweb services get this wrong, so we
                     # workaround the issue and warn the user
                     # rather than raising an error.
-                    logger.error(
+                    logger.warn(
                         'value of data element "{}" with VR Person Name (PN) '
                         'is not formatted correctly'.format(tag)
                     )
                     elem_value.append(v)
                 else:
-                    elem_value.extend(list(v.values()))
-            if vm == '1':
+                    comps = []
+                    if 'Alphabetic' in v:
+                        comps.append(v['Alphabetic'])
+                    if 'Ideographic' in v:
+                        comps.append(v['Ideographic'])
+                    if 'Phonetic' in v:
+                        comps.append(v['Phonetic'])
+                    elem_value.append('='.join(comps))
```

Comment:
Do these not still require an "=" delimiter if missing?  E.g. if ideographic not there but phonetic is.

Reply:
Yes, you are right. I only thought about the json that pydicom generates itself, where this will not happen, but there could be other data of course.

Reply:
Ok, done.